README.md

```diff
@@ -1,38 +1,50 @@
-    - This Unique Id could be used in another table to be linked to the postcode of the house, number of people in household etc...
-    - Hence we feel it would be appropriate to make use of an SQL database, as the data is related in someway.
+   
+   - As you can above the table has 3 columns with the middle column referring to the the unique id provided to the house owner on a particlar street in the neighbourhood.
+   - This Unique Id could be used in another table to be linked to the postcode of the house, number of people in household etc...
+   - Hence we feel it would be appropriate to make use of an SQL database, as the data is related in someway.
+
 
 ### Very basic Schema example of the SQL data and how it links
 
+
 [API-Documentation.png](https://postimg.cc/Lnj1jbqr)
 
+
 ## Consideration of the Requests the API should be capable of handling:
 
+
 - Every request to the neighbourhood API should be a GET request as the system is to only track people, houses and addresses in the neighbourhood.
 - GET people living in the area and their information regarding age and name, the houses and addresses including post code, number of occupants in a house and owner of each house.
 - There weren't be any changes that are made unless a resident moves out of the house, in which case we will use UPDATE and DELETE in order to update the records.
-- POST request will send new data to the API to update the residential data whilst the DELETE request will remove data through the API.
+- PUT request will send new data to the API to update the residential data whilst the DELETE request will remove data through the API.
+
 
 ## Table of HTTP Verbs, Path and Action:
-| HTTP verb      | Path                    | Action    |
-| ------------- |:------------------------:| ---------:|
-| GET           | /neighborhood            | index     |
-| GET           | /neighborhood/house      |   house   |
-| GET           | /neighbourhood/residents | residents |
-| DELETE        | /neighbourhood/residents |   destroy |
-| DELETE        | /neighbourhood/house     |   destroy |
-| POST          | /neighbourhood/house     | create    |
-| POST          | /neighbourhood/residents | create    |
+| HTTP verb     | Path                      | Action   |
+| ------------- |:---------------------------:| ---------:|
+| GET   | /neighborhood          | index    |
+| GET   | /neighborhood/house        |  house  |
+| GET   | /neighborhood/house:{householdsizeid}   |  house  |
+| GET     | /neighbourhood/residents  | residents |
+| GET     | /neighborhood/resident:{ageid}     | resident age |
+| DELETE       | /neighbourhood/residents:id   |  destroy |
+| DELETE       | /neighbourhood/house:id       |  destroy |
+| PUT         | /neighbourhood/house:id    | create   |
+| PUT         | /neighbourhood/residents:id | create   |
```

Comment:
Would have been nice to see more examples of requests. What does the body of a post request need to be successful? Request parameters?